### PR TITLE
Custom type with Combine(). Fix for #3781

### PR DESCRIPTION
**Pull request by BMBurstein**

### CODE REVIEW

File: googletest/include/gtest/internal/gtest-param-util.h [hunk 1]
```diff
@@ -950,6 +950,77 @@ class CartesianProductHolder {
   std::tuple<Gen...> generators_;
 };
 
+template <typename From, typename To>
+class ParamGeneratorConverter : public ParamGeneratorInterface<To> {
```
Comment (ProKashif) [inline googletest/include/gtest/internal/gtest-param-util.h:954]: IMO
```suggestion
class ParamGeneratorConversion: public ParamGeneratorInterface<To> {
```

File: googletest/test/googletest-param-test-test.cc [hunk 2]
```diff
@@ -51,6 +51,7 @@ using ::std::vector;
 using ::testing::AddGlobalTestEnvironment;
 using ::testing::Bool;
 using ::testing::Combine;
+using ::testing::ConvertGenerator;
```
Comment (ProKashif) [inline googletest/test/googletest-param-test-test.cc:54]:
```suggestion
using ::testing::ConversionGenerator;
```

**Review by BugBasherTony — APPROVED** [review]

**Comment by BMBurstein** [comment]
@asoffer @derekmauro I understand that this is not a very high priority, but is there anything that I can do to try to get this merged (or officially rejected) sooner?

**Comment by BMBurstein** [comment]
Thank you

**Comment by asoffer** [comment]
Again, sorry for the delay. I've been having trouble pulling this in. could you sync this to head and I'll try again?

**Comment by BMBurstein** [comment]
@asoffer synced

**Pull request merged**